test/test_settings.py

```diff
@@ -1,7 +1,9 @@
-LOCAL_MQTT_BROKER_ADDRESS = "broker.hivemq.com"
+LOCAL_MQTT_BROKER_ADDRESS = "localhost"
+LOCAL_MQTT_BROKER_PORT = 1883
 LOCAL_MQTT_USER_NAME = None
 
-GW_MQTT_BROKER_ADDRESS = "mqtt.eclipseprojects.io"
+GW_MQTT_BROKER_ADDRESS = "localhost"
+GW_MQTT_BROKER_PORT = 18831
```

Comment:
can you update `template_settings` to have the `GW_MQTT_BROKER_ADDRESS` and `LOCAL_MQTT_BROKER_ADDRESS`  as well?

Reply:
Yes.

Reply:
Done